### ISSUE TITLE
7.0 update?

### ISSUE THREAD

**Issue by Dajova** [yomishino/FFXIVActionEffectRange#16]
Do we have a eta on when this is gonna get updated or is it abandoned?

**Comment by Theletha** [comment]
+1

**Comment by SpringsTea** [comment]
I still love your mod

**Comment by olee** [comment]
Any update on this? I really love this mod and would love to be able to use it again.
Could also try to update this mod myself, but would have to look into modding setup first Oo

**Comment by Theletha** [comment]
Still out here waiting

**Comment by RenNagasaki** [comment]
@Dajova @Theletha @SpringsTea @olee 
I updated it to Dalamud API13 and created a pull request. But since this seems abandoned you may want to look here:

https://github.com/RenNagasaki/FFXIVActionEffectRange

**Comment by SpringsTea** [comment]
@RenNagasaki Do you plan on updating your fork for the dawntrail classes?

**Comment by RenNagasaki** [comment]
If that is necessary, yes. My understanding so far was that it would work out of the box.

**Comment by SpringsTea** [comment]
I don't get any range effects on VPR

**Comment by RenNagasaki** [comment]
Good to know. I'm gonna take a look. Did you add my plugin repo and install my version?

**Comment by SpringsTea** [comment]
Yes I tried your fork out yesterday. It worked for SGE and WAR but not VPR

**Comment by RenNagasaki** [comment]
Gonna fix that

**Comment by RenNagasaki** [comment]
@SpringsTea so I took a look and at least Steal Maw and Reaving Maw work with my version of the plugin.
Just so we are on the same page. You added: https://raw.githubusercontent.com/RenNagasaki/MyDalamudPlugins/master/pluginmaster.json to your third party repositories in dalamud and installed ActionEffectRange from there?

**Comment by SpringsTea** [comment]
Aaah it's my settings from the old version, I didn't realize they carried over.
I had harmful actions unchecked.

Were good 👍

**Comment by Theletha** [comment]
awesome work man!
